waffle/jinja.py

```diff
@@ -15,9 +15,11 @@ def inline_wafflejs_helper(context):
     return _generate_waffle_js(context['request'])
 
 
-jingo.env.globals['waffle'] = {
-    'flag': flag_helper,
-    'switch': switch_is_active,
-    'sample': sample_is_active,
-    'wafflejs': inline_wafflejs_helper
-}
+class WaffleExtension(Extension):
```

Comment:
:+1: I like this approach. Much cleaner than what I came up with. I guess it would need to be tested with django-jinja but I don't see why it wouldn't work.